ext/openssl/tests/openssl_pkcs7_verify_basic.phpt

```diff
@@ -36,6 +42,11 @@ if (file_exists($contentfile)) {
 	echo "true\n";
 	unlink($contentfile);
 }
+
+if (file_exists($pkcsfile)) {
```

Comment:
Pls move it to the `--CLEAN-- ` section (see openssl_x509_export_to_file_basic.phpt for example...)